psa-crypto-sys/build.rs

```diff
@@ -120,70 +122,74 @@ mod common {
         }
     }
 
-    pub fn generate_mbed_crypto_bindings(mbed_include_dir: String) -> Result<()> {
+    pub fn generate_mbed_crypto_bindings(
+        mbed_include_dir: String,
+        external_mbedtls: bool,
+    ) -> Result<()> {
         let header = mbed_include_dir.clone() + "/psa/crypto.h";
 
+        println!("using mbedtls include directory of: {}", mbed_include_dir);
         println!("cargo:rerun-if-changed={}", header);
 
         let out_dir = env::var("OUT_DIR").unwrap();
 
-        #[cfg(not(feature = "prefix"))]
-        let shim_bindings = bindgen::Builder::default()
+        // Common shim builder settings
+        let mut shim_builder = bindgen::Builder::default()
             .clang_arg(format!("-I{}", out_dir))
-            .clang_arg("-DMBEDTLS_CONFIG_FILE=<config.h>")
             .clang_arg(format!("-I{}", mbed_include_dir))
             .header("src/c/shim.h")
             .blocklist_type("max_align_t")
             .generate_comments(false)
-            .size_t_is_usize(true)
-            .generate()
-            .map_err(|_| {
-                Error::new(
-                    ErrorKind::Other,
-                    "Unable to generate bindings to mbed crypto",
-                )
-            })?;
+            .size_t_is_usize(true);
 
         #[cfg(feature = "prefix")]
-        let shim_bindings = bindgen::Builder::default()
-            .clang_arg(format!("-I{}", out_dir))
-            .clang_arg("-DMBEDTLS_CONFIG_FILE=<config.h>")
-            .clang_arg(format!("-I{}", mbed_include_dir))
-            .header("src/c/shim.h")
-            .blocklist_type("max_align_t")
-            .generate_comments(false)
-            .size_t_is_usize(true)
-            .parse_callbacks(Box::new(RenameCallbacks {}))
-            .generate()
-            .map_err(|_| {
-                Error::new(
-                    ErrorKind::Other,
-                    "Unable to generate bindings to mbed crypto",
-                )
-            })?;
+        {
```

Comment:
nit: Only one line, so maybe we can remove the curly braces?

Reply:
Seeing the comparison I think the change you applied is in a different line that I intended. I was suggesting replacing
```
        #[cfg(feature = "prefix")]
        {
            shim_builder = shim_builder.parse_callbacks(Box::new(RenameCallbacks {}));
        }
```

with
```
        #[cfg(feature = "prefix")]
         shim_builder = shim_builder.parse_callbacks(Box::new(RenameCallbacks {}));
```

Reply:
That moment you seek to the wrong line in vim haha

Reply:
@tgonzalezorlandoarm that seems to result in:
```bash
Compiling psa-crypto-sys v0.12.0 (/home/bill/workspace/rust-psa-crypto/psa-crypto-sys)
error[E0658]: attributes on expressions are experimental
   --> psa-crypto-sys/build.rs:145:9
    |
145 |         #[cfg(feature = "prefix")]
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #15701 <https://github.com/rust-lang/rust/issues/15701> for more information
    = help: add `#![feature(stmt_expr_attributes)]` to the crate attributes to enable

For more information about this error, try `rustc --explain E0658`.
```

Reply:
mmm okay, nevermind then. Thanks!